harmonica/isostasy.py

```diff
@@ -0,0 +1,69 @@
+"""
+Function to calculate the thickness of the roots and antiroots assuming the
+Airy isostatic hypothesis.
+"""
+import numpy as np
+
+
+def isostasy_airy(
+    topography,
+    density_upper_crust,
+    density_lower_crust,
+    density_mantle,
+    density_water=None,
+):
+    """
+    Computes the thickness of the roots/antiroots using the Airy's hypothesis.
+
+    In Airy's hypothesis of isotasy, the mountain range can be thought of as a
+    block of lithosphere (crust) floating in the asthenosphere. Mountains have
+    roots (:math:`r`), while ocean basins have antiroots (:math:`ar`)
+    [Hofmann-WellenhofMoritz2006]_ .
+    If :math:`T` is the normal thickness of the Earh's crust, :math:`T + r` and
+    :math:`T + ar` are the isostatic Moho at the cotinental and oceanic
+    points respectively.
+
+    On continental points:
+
+    .. math ::
+        r = \frac{\rho_{uc}}{\rho_m - \rho_{lc}} h
+
+    On oceanic points:
+
+    .. math ::
+        ar = \frac{\rho_{lc} - \rho_w}{\rho_m - \rho_{lc}} h
+
+        where :math:`h` is the topography/bathymetry, :math:`rho_m` is the
+        density of the mantle, :math:`rho_w` is the density of the water and
+        :math:`\rho_{uc}` and :math:`\rho_{lc}` are the density of the upper
+        and lower crust respectively.
```

Comment:
@aguspesce is there a reason for using the lower crust density in the oceans? Would it be possible to have a `density_oceanic` as well?

Reply:
Also, the identation of this block might cause trouble with the equation.